.github/workflows/build.yaml

```diff
@@ -101,3 +106,33 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         run: |
           mvn deploy $SKIP_FLAGS_ALL_TESTS -Dmaven.install.skip=true --file service/pom.xml
+      - name: Publishing Test Results - Checkstyle
+        if: always()
```

Comment:
`always()` means to run this step `always` even if previous steps fail (which is the xecpected behaviour, we want to see the test report even if tests fail)